README.md

```diff
@@ -41,6 +42,11 @@ For installation of Jekyll please follow https://jekyllrb.com/docs/installation/
 
 4. bundle install 
 
+### WSL notes
+If you're running through WSL you need to use the flag `--force-polling` or it will not regenerate automatically on changes.
```

Comment:
Probably good to specify that the `--force-polling` flag is for jekyll

Reply:
I feel like moving it under the jekyll section would make it self explanatory.

Reply:
Somewhat, but since git is mentioned directly above in the windows section, so it's not always obvious

Reply:
With the change you've made, I think it's now fine though